composables/api/useProductList.ts

```diff
@@ -2,28 +2,33 @@ import { useQuery } from 'vue-query';
 import type { Database } from '~/types/generated-types';
 import type { FullProduct } from '~/types/products';
 
-export const useProductList = (count = Infinity) => {
+export const useProductList = (amount = ref(Infinity)) => {
```

Comment:
Passing a `ref` here allowed to refetch the data when we click "see more" on the `/fleet` page